rubocop-rootstrap.gemspec

```diff
@@ -15,8 +13,12 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['LICENSE.txt', 'README.md', 'lib/**/*', 'config/*.yml']
 
-  # Rubocop version is constrained to these versions to exclude breaking changes from previous versions
+  #
+  # Rubocop version is constrained to these versions to exclude
+  # breaking changes from previous versions.
```

Comment:
Weird comment # usage here

Reply:
Suggestions?

Reply:
Nevermind, It just looked weird with some extra jumps in the mobile app, now that I check it in web is perfect